src/commands/addBinding/addBinding.ts

```diff
@@ -36,7 +36,10 @@ export async function addBinding(context: IActionContext, data: Uri | LocalFunct
     } else {
         if (!data) {
             const noItemFoundErrorMessage: string = localize('noLocalProject', 'No matching functions found. C# and Java projects do not support this operation.');
-            data = await ext.rgApi.tree.showTreeItemPicker<LocalFunctionTreeItem>(/Local;ReadWrite;Function;/i, { ...context, noItemFoundErrorMessage });
+            data = await ext.rgApi.pickAppResource<LocalFunctionTreeItem>({ ...context, noItemFoundErrorMessage }, {
```

Comment:
This is going to be in the workspaceProvider, so don't think that this will work.  Since we don't change the context value in the workspace, should be able to just do `ext.rgApi.workspaceTree.showTreeItemPicker`